src/Microsoft.Crank.Jobs.Bombardier/Program.cs

```diff
@@ -382,6 +381,15 @@ public static async Task MeasureFirstRequest(string[] args)
             {
                 Console.WriteLine("A timeout occurred while measuring the first request");
             }
+            catch (HttpRequestException)
+            {
+                Console.WriteLine("A connection exception occurred while measuring the first request");
```

Comment:
Any useful info on HttpRequestException? `Message`?

Reply:
This is the exceptions hierarchy; the message is not really useful per se

```
[STDERR] Unhandled exception. System.Net.Http.HttpRequestException: An error occurred while sending the request.
[STDERR]  ---> System.IO.IOException: Unable to read data from the transport connection: Connection reset by peer.
[STDERR]  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
```